raiden/tests/integration/test_send_queued_messages.py

```diff
@@ -165,25 +166,55 @@ def test_payment_statuses_are_restored(
 
     app0.event_handler = HoldRaidenEvent()
     app0.event_handler.hold(SendSecretReveal, {})
+    app2.event_handler = HoldRaidenEvent()
+    app2.event_handler.hold(SendSecretRequest, {})
```

Comment:
I don't think this is necessary, `app0` is not sending the secret reveal anyways

Reply:
It's sending it when it restarts I believe since it no longer has the `HoldRaidenEvent` handler